### PR TITLE
Add hljs.PHRASAL_WORDS_MODE to all comment modes

### PR DESCRIPTION
**Pull request by Sannis**
I'm not think that this is so needed, but...

@isagalaev, what do you think about this?

### CODE REVIEW

**Comment by Sannis** [comment]
If `hljs.COMMENT()` will be accepted, I'll make same thing for numbers - most of that should have zero relevance to avoid wrong detection.

File: src/highlight.js [hunk 1]
```diff
@@ -669,21 +669,23 @@ https://highlightjs.org/
...
+      className: 'comment',
+      begin: begin, end: end
+    };
+    if (inherits) {
+      mode = hljs.inherit(mode, inherits);
+    }
+    if (!mode.contains) {
+      mode.contains = [];
+    }
+    mode.contains.push(hljs.PHRASAL_WORDS_MODE);
```
Comment (isagalaev) [inline src/highlight.js]: These four lines could be spelled shorter: `mode.contains = (mode.contains || []).push(hljs.PHRASAL_WORDS_MODE);`

**Comment by isagalaev** [comment]
Merged it regardless of code in hljs.COMMENT, as it is ultimately a good change anyway. We could refactor stuff right in master.

**Comment by isagalaev** [comment]
Thanks a lot!

**Comment by Sannis** [comment]
Nice, will do TODO and doctag highlighting as next step.

**Pull request merged**